packages/dds/merge-tree/src/properties.ts

```diff
@@ -69,21 +69,31 @@ export function combine(
 }
 
 export function matchProperties(a: PropertySet | undefined, b: PropertySet | undefined) {
+	let entriesA;
+	let entriesB;
+	if (!a && !b) {
+		return true;
+	}
 	if (a) {
-		if (!b) {
+		entriesA = Object.entries(a);
```

Comment:
i wonder which is cheaper, object.keys or object.entries, my guess is object.keys